tests/cases/uint/uint_try_from_felt.cairo

```diff
@@ -0,0 +1,18 @@
+use traits::TryInto;
+
+// u128 doesn't have this functionality yet
```

Comment:
Yes, it does: https://github.com/starkware-libs/cairo/blob/72d6f0638d3a7e893e4e106d3eb03a613a82b5c7/corelib/src/integer.cairo#L34